website/source/docs/agent/options.html.md

```diff
@@ -1036,7 +1036,9 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="recursors"></a><a href="#recursors">`recursors`</a> This flag provides addresses of
   upstream DNS servers that are used to recursively resolve queries if they are not inside the service
   domain for Consul. For example, a node can use Consul directly as a DNS server, and if the record is
-  outside of the "consul." domain, the query will be resolved upstream.
+  outside of the "consul." domain, the query will be resolved upstream. As of Consul 1.0.1 recursors
+  can be provided as ip addresses or as go-sockaddr templates. The list of ip addresses is deduplicated
```

Comment:
For the last sentence I would rewrite it as "IP addresses are resolved in order, and duplicates are ignored".